server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JMAPModule.java

```diff
@@ -207,7 +207,7 @@ JMAPDraftConfiguration provideDraftConfiguration(PropertiesProvider propertiesPr
                 .keystoreType(configuration.getString("tls.keystoreType", null))
                 .secret(configuration.getString("tls.secret", null))
                 .jwtPublicKeyPem(loadPublicKey(fileSystem, ImmutableList.copyOf(configuration.getStringArray("jwt.publickeypem.url"))))
-                .authenticationStrategies(Optional.ofNullable(configuration.getList(String.class, "authentication.strategy.draft", null)))
+                .authenticationStrategies(Optional.ofNullable(ImmutableList.copyOf(configuration.getStringArray("authentication.strategy.draft"))))
```

Comment:
The Optional.ofNullable is not needed...

Reply:
Optional.of then?

Reply:
Maybe not, as it handled the not present case.

I think:

```
Optional.ofNullable(configuration.getStringArray("authentication.strategy.draft")).map(ImmutableList::copyOf)
```

is better as it preserves nullity.

Also this shows we can benefit from unit tests on this conf parsing. You can get a look at what was done for `WebAdminConfiguration`.

Reply:
Yeah will do later

Reply:
> is better as it preserves nullity.

Well the failures in the tests (and when debugging them) shows that it doesn't... will find an other way

Reply:
We always end up with a Some([]) with this... while the initial code actually is ok on the nullity. I tried as well to add the auth strategies in jmap.properties in integration tests for jmap rfc8621 and ran AuthenticationContract and it's green. 

Maybe it's what i did with Helm, or I messed up somewhere else... Will test deeper when I have time